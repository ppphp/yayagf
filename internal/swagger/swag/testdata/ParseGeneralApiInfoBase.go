package main

// @host petstore.swagger.io
// @basepath /v2
// @schemes http https

// @x-google-endpoints [{"name":"name.endpoints.environment.cloud.goog","allowCors":true}]
// @x-google-marks "marks values"

func main() {}
