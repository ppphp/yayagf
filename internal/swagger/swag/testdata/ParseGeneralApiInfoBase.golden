{
    "schemes": [
        "http",
        "https"
    ],
    "swagger": "2.0",
    "info": {
        "contact": {},
        "license": {}
    },
    "host": "petstore.swagger.io",
    "basePath": "/v2",
    "paths": {},
    "x-google-endpoints": [
        {
            "allowCors": true,
            "name": "name.endpoints.environment.cloud.goog"
        }
    ],
    "x-google-marks": "marks values"
}